types/waf.go

```diff
@@ -133,7 +133,7 @@ func ParseRequestBodyLimitAction(rbla string) (RequestBodyLimitAction, error) {
 	return -1, fmt.Errorf("invalid request body limit action: %s", rbla)
 }
 
-type auditLogPart byte
+type AuditLogPart byte
```

Comment:
Does this classify as API change?

Reply:
only if we make it private

Reply:
Could you elaborate a bit on why this becomes public?